### DISCUSSION TITLE
Blog post working fine locally but not showing in github pages

### DISCUSSION THREAD

**Discussion by s4bot3ur** [razonyang/hugo-theme-bootstrap#1078] [Q&A] [answered]
I recently added a new series to my blog website, but it initially didn’t show up. After I included the _index.md file in the content, it began appearing on the homepage. However, it's still not displaying in the categories, tags, or series sections. Can someone please help me with this

**Comment by razonyang** [comment]
It's difficult to assist without a reproducer.

- How do you create content?
- Is the content in `draft` state?
- What is the content?
- What do you expect?

It would be helpful if you could provide the repo to reproduce.

**Comment by s4bot3ur** [comment]
I’m experiencing an issue with my blog website (s4bot3ur.github.io). On the homepage, there’s a write-up about "NavieReceiver," and I have created a new series called "Damn-Vulnerable-Defi."

However, when I navigate to the "Blogs" section and then click on "Series," the "Damn-Vulnerable-Defi" series does not appear. Interestingly, I can access the series directly by entering the URL: https://s4bot3ur.github.io/series/damn-vulnerable-defi/.

Could you please help me identify why the series is not showing in the navigation?

**Reply by razonyang** [reply]
OK, it's very clear now. I'll check later. Cloning is slow on my network.

**Comment by razonyang** [comment, marked as answer]
Hmm, it seems like there's something wrong with the deployment, as the build results are different between local and GitHub workflow.

| Local | GitHub Workflow |
| --- | --- |
| ![image](https://github.com/user-attachments/assets/f255f3f7-029d-44ab-be4e-470b330faf6d) | ![image](https://github.com/user-attachments/assets/382f1737-6a5e-4699-8083-698eff63dc3c) |

The page count does not match.

To diagnose, I'd suggest removing `public` and `resources` folder from repo, those folders are no need to commit to repo.

To do so,

```sh
git rm -r --cache public resources
```

And append those two folder into `.gitignore` file.

```git
// .gitignore
node_modules
public/
resources/
```

And then commit changes and push it to remote.

```sh
git add .gitignore
git commit -m 'Remove the public folder and resources folder'
git push origin main
```

And check if the result of re-deploy later.

**Reply by razonyang** [reply]
I just tested it and it should work after removing those folders. The new build is the same as the local build on my fork.

Changes: https://github.com/razonyang/s4bot3ur.github.io/commit/f9d290c86b53b85fe43855dce53ef18283a8715c
Deploy Logs: https://github.com/razonyang/s4bot3ur.github.io/actions/runs/11558521076/job/32171090109#step:7:1

![image](https://github.com/user-attachments/assets/b5757fa7-7772-4e8b-951f-e8c59a81851b)

**Comment by s4bot3ur** [comment]
It is working now. Thank you so much for the help.

**Comment by s4bot3ur** [comment]
I’m experiencing an issue with my blog website (s4bot3ur.github.io). On the homepage, I have created a new series names War-Games-Ctf. the files are added to git and those are visible in main branch :[blogs]( https://github.com/s4bot3ur/s4bot3ur.github.io/tree/main/content/blogs)

But these files are not displayed in the website

Could you please help me identify why the series is not showing in the website

**Reply by razonyang** [reply]
There are two issues on your content:

1. The `draft` was set as `true`, remove it or set as `false` to publish the post.
2. The `series` should be an array, for example `series="War-Games-CTF"` should be `series=["War-Games-CTF"]`, the build won't fail, but I'd recommend declaring it as an array.

**Reply by s4bot3ur** [reply]
Now it is working fine. Thank you for the help.

**Reply by razonyang** [reply]
No problem.
